app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    enum :role, [ :user, :admin ]
```

Comment:
Can't we define it as `enum role:  { user: 0, admin: 1 }` ?
I am curious about the category of the user
user => who will buy tickets, right?
admin => who will sell tickets?
Then don't you think for bus ticketing systems, we would be needing an authority who would monitor/have access to everything and an agent as a role who would actually sell the tickets?
I am actually sharing my personal thought here... 😅 
correct me if i am wrong and feel free to defend my questions.

Reply:
Role field updated.
we didn't add agent role as our system is totally online where admin is responsible for adding all required resources and user can only buy tickets through this web app.

Reply:
ahhh okay!
makes sense. Maybe i was thinking more in our Bangladeshi perspective then. Thanks.